configuration/items.md

```diff
@@ -202,17 +202,34 @@ Two naming schemes are established in the community for Group names:
 {: #label}
 ### Label
 
-Label text is used to describe an Item in a human-readable way.
-Graphical UIs will display the label text when the Item is included, e.g. in [Basic UI]({{base}}/addons/uis/basic/readme.html) in a [Sitemap]({{base}}/configuration/sitemaps.html) definition.
+The label is used to describe an Item in a human-readable way.
+Graphical UIs will display the label when the Item is included, e.g. in [Basic UI]({{base}}/addons/uis/basic/readme.html) in a [Sitemap]({{base}}/configuration/sitemaps.html) definition.
 Some I/O services (e.g. the Amazon Alexa skill) also use the label to match an external voice command to an Item.
 
-In textual configurations the label, in quotation marks, appears next to the optional state presentation field in square brackets (see below).
-The label for the Item in the following example is "Temperature":
+In textual configurations the label in quotation marks, contains the label text and the optional state presentation field in square brackets (see below).
+They will be showed side by side, the text left aligned and the state representation right-aligned.
+The label text for the Item in the following example is "Temperature" and the optional state representation is set to be displayed e.g as "23.9 °C":
 
 ```java
 Number Livingroom_Temperature "Temperature [%.1f °C]"
 ```
 
+The label defined for an Item can be overwritten by the label definition on a sitemap.
+If the label definition on the Item contained label text and the state representation these have to be considered separatly.
+
+```perl
+sitemap demo label="My home automation" {
+    Frame label="Date" {
+        // Overrides label text only, keeps state representation defined by Item
+        Text item=Livingroom_Temperature label="Livingroom"
+        // Overrides label text and sets state representation to be empty, not displayed
```

Comment:
Suggestion:
Overrides the text and hides any state representation.

Reply:
Sadly this is not the case. There are at least cases in which a state will be shown. You need to explicitly define `[]` if you want an empty representation.

This overwrite is good for just defining a custom label text in sitemap

Reply:
Isn't this the commen for line 226 whre the `[]` are explicitly defined?
Or am getting your comment wrong now?!

Reply:
```
String Whg_Light_Preset "Wohnung Beleuchtung [%s]"
Switch item=Whg_Light_Preset icon="light" label="Test"
```

![grafik](https://user-images.githubusercontent.com/2870104/42460739-f3e7c2b4-839e-11e8-81d1-f6585143e047.png)

Reply:
You are totally right!!
Sorry @Josar, please ignore my critique here. You did an excellent job finding all cases 😅